test/naming_tests.jl

```diff
@@ -57,3 +57,75 @@ s = savename(e; allowedtypes = (Any,), expand = ["c"])
 @test ')' ∈ s
 @test occursin("a=3", s)
 @test occursin("b=4", s)
+
+# Tests for parse_savename
+
+function test_convert(prefix::AbstractString,c;kwargs...)
+    name = savename(prefix,c;kwargs...)
+    prefix, _b, suffix = DrWatson.parse_savename(name)
+    dicts_equal(_b,c) && prefix == prefix && suffix == ""
+end
+
+function test_convert(c,suffix::AbstractString;kwargs...)
+    name = savename(c,suffix;kwargs...)
+    prefix, _b, suffix = DrWatson.parse_savename(name)
+    dicts_equal(_b,c) && suffix == suffix && prefix == ""
```

Comment:
same comment here for `suffix`.

Maybe I have misunderstood something, so please correct me if need be.